examples/storybook/src/wrap-with-hits.ts

```diff
@@ -12,6 +12,7 @@ type WrapWithHitsParams = {
   methods?: {};
   searchFunction?: (helper: Helper) => void;
   searchClient?: {};
+  insightsClient?: (method: string, payload: object) => void;
```

Comment:
We can add a `@TODO` comment to not forget to update type with the one from InstantSearch once available.

Reply:
👍 it's been added